--- muddleftpd-1.3.13.1/doc/Makefile.in~	2003-11-02 17:51:55.000000000 +0100
+++ muddleftpd-1.3.13.1/doc/Makefile.in	2003-11-09 11:30:00.000000000 +0100
@@ -53,13 +53,13 @@
 	install man/mudpasswd.1 -m 644 $(MANDIR)/man1/$(MUDPASSWDPROG).1
 	mkdir -p $(INFODIR)
 	install $(DOCOBJ) -m 644 $(INFODIR)
-	mkdir -p $(DOCDIR)/doc/muddleftpd
-	install -m 644 cookie.txt $(DOCDIR)/doc/muddleftpd/cookie.txt
-	install -m 644 ftpcmds.txt $(DOCDIR)/doc/muddleftpd/ftpcmds.txt
-	install -m 644 muddleftpd.txt $(DOCDIR)/doc/muddleftpd/muddleftpd.txt
-	install -m 644 mudlogd.txt $(DOCDIR)/doc/muddleftpd/mudlogd.txt
-	install -m 644 mudpasswd.txt $(DOCDIR)/doc/muddleftpd/mudpasswd.txt
-	install -m 644 reference.txt $(DOCDIR)/doc/muddleftpd/reference.txt
+#	mkdir -p $(DOCDIR)/doc/muddleftpd
+#	install -m 644 cookie.txt $(DOCDIR)/doc/muddleftpd/cookie.txt
+#	install -m 644 ftpcmds.txt $(DOCDIR)/doc/muddleftpd/ftpcmds.txt
+#	install -m 644 muddleftpd.txt $(DOCDIR)/doc/muddleftpd/muddleftpd.txt
+#	install -m 644 mudlogd.txt $(DOCDIR)/doc/muddleftpd/mudlogd.txt
+#	install -m 644 mudpasswd.txt $(DOCDIR)/doc/muddleftpd/mudpasswd.txt
+#	install -m 644 reference.txt $(DOCDIR)/doc/muddleftpd/reference.txt
 
 clean:
 
--- muddleftpd-1.3.13.1/modules/auth/authlibmud/Makefile.in~	2003-11-03 13:19:41.000000000 +0100
+++ muddleftpd-1.3.13.1/modules/auth/authlibmud/Makefile.in	2003-11-09 18:41:21.000000000 +0100
@@ -19,9 +19,9 @@
 
 install: all
 	mkdir -p $(libdir)
-	mkdir -p $(DOCDIR)/doc/muddleftpd
+#	mkdir -p $(DOCDIR)/doc/muddleftpd
 
-	install README -m 644 $(DOCDIR)/doc/muddleftpd/README.authmud
+#	install README -m 644 $(DOCDIR)/doc/muddleftpd/README.authmud
 	install $(LIBNAME) -m 644 $(libdir)
 
 clean: 
--- muddleftpd-1.3.13.1/modules/auth/authlibmysql/Makefile.in~	2003-11-03 13:19:51.000000000 +0100
+++ muddleftpd-1.3.13.1/modules/auth/authlibmysql/Makefile.in	2003-11-09 18:41:43.000000000 +0100
@@ -19,9 +19,9 @@
 
 install: all
 	mkdir -p $(libdir)
-	mkdir -p $(DOCDIR)/doc/muddleftpd
+#	mkdir -p $(DOCDIR)/doc/muddleftpd
 
-	install README -m 644 $(DOCDIR)/doc/muddleftpd/README.authmysql
+#	install README -m 644 $(DOCDIR)/doc/muddleftpd/README.authmysql
 	install $(LIBNAME) -m 644 $(libdir)
 
 
--- muddleftpd-1.3.13.1/modules/auth/authlibsmb/Makefile.in~	2003-11-03 13:20:04.000000000 +0100
+++ muddleftpd-1.3.13.1/modules/auth/authlibsmb/Makefile.in	2003-11-09 18:42:07.000000000 +0100
@@ -22,9 +22,9 @@
 
 install: all
 	mkdir -p $(libdir)
-	mkdir -p $(DOCDIR)/doc/muddleftpd
+#	mkdir -p $(DOCDIR)/doc/muddleftpd
 
-	install README -m 644 $(DOCDIR)/doc/muddleftpd/README.authsmb
+#	install README -m 644 $(DOCDIR)/doc/muddleftpd/README.authsmb
 	install $(LIBNAME) -m 644 $(libdir)
 
 
